release-notes-4.0.0.md

```diff
@@ -1,14 +1,13 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
 
-#### Known Issues
+* caf-common upgraded to [1.20.0-288](https://github.com/CAFapi/caf-common/releases/tag/v1.20.0)
 
 #### Breaking Changes
-*   SCMOD-9988: Lyra client dropped  
+*   [SCMOD-9988](https://portal.digitalsafe.net/browse/SCMOD-9988): Lyra client dropped  
```

Comment:
Remove the JIRA link, this is open source repo so consumers will not be able to access the link.